amf-client/shared/src/test/resources/validations/conversion-exceptions/reports/invalid-root-field-value.report

```diff
@@ -1,5 +1,5 @@
-Model: amf://id
-Profile: AMF Graph
+Model: file://amf-client/shared/src/test/resources/validations/conversion-exceptions/invalid-root-field-value.jsonld
+Profile: 
```

Comment:
Unique/MultiAmfGraphSyamlConversionExceptionTest: report does not have a profile because they are parsing errors, and in this case the profile name is obtained through the sourceVendor. Since this suite parses jsonlds and no sourceVendor is present the ProfileName is set as UnkownProfile.